### DISCUSSION TITLE
brew install Azure/kubelogin/kubelogin fails with "kubelogin: SHA256 mismatch" error

### DISCUSSION THREAD

**Discussion by sankarsekar1985** [Azure/kubelogin#393] [Q&A]
Hello All,

I'm facing the below issue while trying to install kubelogin in mac os.

Any insight would be helpful...

Error: kubelogin: SHA256 mismatch
Expected: f03a1ac8ac2a94d617ef6753b980ba19f63a2f80324c3cf7dc5892b602ef0bf3
  Actual: 61dcff292a024d3dcdc9ee9528616523d6fe93408f0f3252455930cfb64a55cb
    File: /*/b57c7b112cd28fb8bcc56f26ed802a870890f39c8d8b6c9a28ece1053ecd880b--kubelogin-darwin-arm64.zip
To retry an incomplete download, remove the file above.

brew config
HOMEBREW_VERSION: 4.2.3
ORIGIN: https://github.com/Homebrew/brew
HEAD: b3751bca8c4a3c76107d5aa3b75b81db93cf4bb6
Last commit: 3 days ago
Core tap JSON: 10 Jan 18:38 UTC
Core cask tap JSON: 10 Jan 18:38 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.6.2-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: N/A
Rosetta 2: false

Thanks and Regards,
Sankar

**Comment by weinong** [comment]
can you provide full log or verbose log? it worked on my apple silicon Mac.

**Reply by VNBharadwaja** [reply]
I'm using silicon Mac but I'm also facing the same issue.

**Reply by weinong** [reply]
@VNBharadwaja it worked for me. did you run `brew update`?

**Reply by VNBharadwaja** [reply]
@weinong yes I did.
I’m running this on Apple M2 pro. brew is latest version, 4.2.20

**Reply by weinong** [reply]
can you share the log?
